code/datums/status_effects/buffs.dm

```diff
@@ -0,0 +1,84 @@
+/datum/status_effect/xeno_rejuvenate
+	id = "xeno_rejuvenate"
+
+/datum/status_effect/xeno_rejuvenate/on_creation(mob/living/new_owner, set_duration)
```

Comment:
I think these abiltities all use the default icon pls fix

Reply:
fixed

Reply:
added icon